### PR TITLE
Diffuser change upgraded to 0.26.3 along with MLPERF SD XL support

### PR DESCRIPTION
**Pull request by ANSHUMAN87**
1. MLPERF SD XL inference related change unstreamed
2. All Diffuser changes upgraded to 0.26.3

JH...!

### CODE REVIEW

**Comment by piotrbocian** [comment]
@astachowiczhabana please also take a look

File: optimum/habana/diffusers/schedulers/scheduling_euler_discrete.py [hunk 1]
```diff
@@ -257,7 +277,10 @@ def step(
         # 2. Convert to an ODE derivative
         derivative = (sample - pred_original_sample) / sigma_hat
 
-        dt = sigma_next - sigma_hat
+        if self.is_inference:
```
Comment (kumarans-ai) [inline optimum/habana/diffusers/schedulers/scheduling_euler_discrete.py:280]: can we try to remove this flag , keep it as default implementation/optimal implementation, unless it affects any model

**Review by kumarans-ai — APPROVED** [review]

**Comment by astachowiczhabana** [comment]
https://github.com/huggingface/optimum-habana/pull/1135

**Comment by astachowiczhabana** [comment]
https://github.com/huggingface/optimum-habana/pull/1204

**Comment by astachowiczhabana** [comment]
https://github.com/huggingface/optimum-habana/pull/1163/

**Pull request merged**